homeassistant/components/alexa/capabilities.py

```diff
@@ -716,3 +716,34 @@ def configuration(self):
         if code_format == FORMAT_NUMBER:
             return {"supportedAuthorizationTypes": [{"type": "FOUR_DIGIT_PIN"}]}
         return []
+
+
+class AlexaChannelController(AlexaCapibility):
+    """Implements Alexa.ChannelController.
+
+    https://developer.amazon.com/docs/device-apis/alexa-channelcontroller.html
+    """
+
+    def name(self):
+        """Return the Alexa API name of this interface."""
+        return "Alexa.ChannelController"
+
+    def properties_supported(self):
+        """Return what properties this entity supports."""
+        return [{"name": "channel"}]
+
+    def properties_proactively_reported(self):
+        """Return True if properties asynchronously reported."""
+        return False
+
+    def properties_retrievable(self):
+        """Return True if properties can be retrieved."""
+        return False
+
+    def get_property(self, name):
+        """Read and return a property."""
+        if name != "channel":
+            raise UnsupportedProperty(name)
+
+        # Return a temporary empty channel property.
+        return {"number": "1", "callSign": "", "affiliateCallSign": ""}
```

Comment:
I don't think any of this is needed. 

- According to the Alexa API docs it appears the the channel property is not required to be disclosed during discovery, and is only reported in the Alexa.Response context properties in response to directives which is set in each of the handlers. Therefore `get_property` is never called for this controller. 
- Since you're not overriding the `AlexaCapability` Base methods for `properties_retrievable` and `properties_proactively_reported` there is not reason to duplicate the code.

Reply:
I'm aware that there is no need to implement that code (that is not going to be called based on the property you mentioned). I put it just as a template in case we'll add the feature to retrieve the current channel.